app/settings.py

```diff
@@ -28,7 +28,12 @@
 FTP_HEARTBEAT_FOLDER = os.getenv('FTP_HEARTBEAT_FOLDER', '/heartbeat')
 
 RABBIT_QUEUE = os.getenv('RABBITMQ_QUEUE', 'sdx-survey-notifications')
+RABBIT_DELAY_QUEUE = os.getenv('RABBIT_DELAY_QUEUE', 'sdx-delayed-survey-notifications')
```

Comment:
If we're being consistent then don't we also either need to drop the use of "sdx" at the start of the string here and get rid of hyphens?